packages/expo-store-review/ios/EXStoreReview/EXStoreReviewModule.m

```diff
@@ -7,15 +7,15 @@ @implementation EXStoreReviewModule
 
 UM_EXPORT_MODULE(ExpoStoreReview);
 
-- (NSDictionary *)constantsToExport
+UM_EXPORT_METHOD_AS(isAvailableAsync,
+                    isAvailableAsync:(UMPromiseResolveBlock)resolve
+                            rejecter:(UMPromiseRejectBlock)reject)
 {
-  BOOL isSupported;
   if (@available(iOS 10.3, *)) {
-    isSupported = [SKStoreReviewController class] ? YES : NO;
```

Comment:
i think we should change this back because it is possible for folks to use this library and forget to link the StoreKit framework, in which case this would be unavailable even though they're on iOS >= 10.3